delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/errors/files/FileErrors.scala

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.bluebrain.nexus.testkit.errors.files
+
+import ch.epfl.bluebrain.nexus.testkit.CirceLiteral.circeLiteralSyntax
+
+object FileErrors {
```

Comment:
It feels a bit weird to have this here and the testkit being aware of a plugin.
Even if storage should not be a plugin imho.
And the remote container is in this module too

Reply:
Hum, yeah. I have merged this one but perhaps we can rework this. It could just be moved to the storage plugin